### PR TITLE
Feature/docs edit

### PR DESCRIPTION
**Pull request by JessBloem**
## Pull Request

**Description:**

Added demo 3 section to the docs 

**Related Issues:**

[Reference any related issues or documents.]

**Type of Change:**

- [ ] Bug Fix
- [ ] New Feature
- [ ] Enhancement
- [ ] Refactoring
- [x] Documentation Update

**Checklist:**

- [ ] My code follows the coding standards of this project.
- [ ] I have performed a self-review of my code.
- [ ] I have added/updated relevant documentation.
- [ ] My changes generate no new warnings or errors.
- [ ] I have tested my changes.

**Screenshots (if applicable):**

[Add screenshots to help reviewers understand the changes visually.]

**Reviewers:**

- @BlockchainDlamini 

**Additional Notes:**

[Any additional information that may be helpful for reviewers.]

### CODE REVIEW

File: docs/.vuepress/.cache/deps/_metadata.json
Comment (BlockchainDlamini) [inline docs/.vuepress/.cache/deps/_metadata.json:1]: is this file important?, is it meant to be private or can it be exposed?

**Review by BlockchainDlamini — APPROVED** [review]
Other than my one comment, I think you're safe to merge

File: docs/.vuepress/.cache/deps/_metadata.json
Comment (JessBloem) [reply docs/.vuepress/.cache/deps/_metadata.json:1]: I think this is part of the vuepress stuff. When I set up the docs, their documentation did not say that any of the files have to be private. I am not sure though.

Comment (JessBloem) [reply docs/.vuepress/.cache/deps/_metadata.json:1]: It seems like this file changes each time I run it

**Comment by JessBloem** [comment]
> Other than my one comment, I think you're safe to merge

I am not sure if that file should be private or public. It seems to change every time I run it, so it might be fine if it is public

**Pull request merged**